dwdatareader/__init__.py

```diff
@@ -170,15 +170,17 @@ def series_generator(self, chunk_size):
                 'DWGetScaledSamplesCount({})={} should be non-negative'.format(
                     self.index, count))
 
+        data = numpy.empty(chunk_size, dtype=numpy.double)
+        time = numpy.empty_like(data)
         for chunk in range(0, count, chunk_size):
-            data = numpy.empty(chunk_size, dtype=numpy.double)
-            time = numpy.empty_like(data)
-            stat = DLL.DWGetScaledSamples(self.index, chunk, chunk_size,
-                                          data.ctypes, time.ctypes)
+            chunk_size = min(chunk_size, count - chunk)
+            stat = DLL.DWGetScaledSamples(self.index,
+                    chunk, chunk_size,
```

Comment:
Nitpick: align args to DLL.DWGetScaledSamples according to PEP8

Reply:
Thanks! I haven't been watching for that one.